packages/admin/dashboard/src/routes/orders/order-detail/components/order-summary-section/order-summary-section.tsx

```diff
@@ -380,15 +375,17 @@ const Item = ({
 }: {
   item: AdminOrderLineItem
   currencyCode: string
-  reservation?: ReservationItemDTO | null
+  reservation?: AdminReservation
   returns: AdminReturn[]
   claims: AdminClaim[]
   exchanges: AdminExchange[]
 }) => {
   const { t } = useTranslation()
+
   const isInventoryManaged = item.variant?.manage_inventory
   const hasInventoryKit =
     isInventoryManaged && (item.variant?.inventory_items?.length || 0) > 1
+  const hasUnfulfilledItems = item.quantity - item.detail.fulfilled_quantity > 0
```

Comment:
Are both fields guaranteed to be set?

Reply:
yup, they are always returned on `order.items`